tests/suites/0_stateless/03_dml/03_0003_select_group_by.sql

```diff
@@ -41,3 +41,11 @@ select created_at, sum(count) from t_datetime group by created_at order by creat
 select created_time, sum(count) from t_datetime group by created_time order by created_time;
 
 drop table t_datetime;
+
+--
+SELECT count(*) FROM numbers_mt(10000000) group by number limit 5;
+set group_by_two_level_threshold=10;
+SELECT count(*) FROM numbers_mt(10000000) group by number limit 5;
+set group_by_two_level_threshold=1000000000;
+SELECT count(*) FROM numbers_mt(10000000) group by number limit 5;
```

Comment:
add `order by number ` to keep the result stable.

Reply:
the result seems stable without order by.

Reply:
Yes, it happens, but if we run the query in different env(max_thread is different), it may be not stable